exercises/practice/accumulate/AccumulateTests.vb

```diff
@@ -7,12 +7,12 @@ Public Class AccumulateTest
         Assert.Equal(New Integer() {}.Accumulate(Function(x) x * x), New Integer() {})
     End Sub
 
-    <Fact(Skip := "Remove this Skip property to run this test")>
+    <Fact(Skip:="Remove this Skip property to run this test")>
```

Comment:
I've normalized this across exercises as otherwise the CI script did not correctly replace it